pkgconfig/pkgconfig.py

```diff
@@ -110,6 +110,38 @@ def libs(package):
     return _query(package, '--libs')
 
 
+def variables(package):
+    """Return a dictionary of all the variables defined in the .pc pkg-config
+     file of 'packae'"""
+
+    if exists(package):
+
+        pkg_config_exe = os.environ.get('PKG_CONFIG', None) or 'pkg-config'
+
+        # get the list of all the variables defined in the .pc file
+        cmd = '{0} {1} {2}'.format(
+            pkg_config_exe, '--print-variables', package)
+
+        proc = Popen(shlex.split(cmd), stdout=PIPE, stderr=PIPE)
+        out, _ = proc.communicate()
+        _variables = filter(lambda x: x != '', out.decode('utf-8').split('\n'))
+
+        # get the variable values
+        retval = dict()
+        for variable in _variables:
+            cmd = '{0} --variable={1} {2}'.format(
+                pkg_config_exe, variable, package)
+            proc = Popen(shlex.split(cmd), stdout=PIPE, stderr=PIPE)
+            out, _ = proc.communicate()
+            retval[variable] = out.decode('utf-8').strip()
+
+        return retval
+    else:
```

Comment:
no problem. I personally don't mind, because it just one level of indentation. But it is fine.